core/src/main/kotlin/ch/ergon/dope/resolvable/clause/model/UnsetClause.kt

```diff
@@ -0,0 +1,32 @@
+package ch.ergon.dope.resolvable.clause.model
+
+import ch.ergon.dope.DopeQuery
+import ch.ergon.dope.resolvable.clause.IUpdateUnsetClause
+import ch.ergon.dope.resolvable.expression.unaliased.type.Field
+import ch.ergon.dope.resolvable.formatToQueryString
+import ch.ergon.dope.validtype.ValidType
+
+class UnsetClause(
+    field: Field<out ValidType>,
+    private val fields: MutableList<Field<out ValidType>> = mutableListOf(),
```

Comment:
instead of this mutable list you could just make it vararg and change the function `unset` to also have vararg fields.

Reply:
done